Terminal resizing in the UI works for remote clusters

kontsevoy opened the pull request:
Fixes #778 

The PR may appear big because of all the docker stuff, but in reality I just added a proper processing of "site" URL argument and the back-end uses a different `auth.ClientI` pointer to talk to a proper auth API.

kontsevoy committed:
Terminal resizing in the UI works for remote clusters

Fixes #778

kontsevoy committed:
Merge branch 'master' into ev/clusters

klizhentas commented on lib/web/sessions.go:
```diff
@@ -65,18 +65,19 @@ func (c *SessionContext) getTerminal(sessionID session.ID) (*terminalHandler, er
 	})
 	if err != nil {
-		return trace.Wrap(err)
+		log.Error(err)
```
I think I understand why fallthrough is necessary, but can you add comment for future readers?

kontsevoy replied:
well... it's not a critical error. no need to abort the function, quite common. frankly, I would even stick it into `_` and not even bother... I'll add a comment.

kontsevoy replied:
done

kontsevoy committed:
Minor fix...

Replaced raw string with a proper constant (which fixed the regression
with the client IP being incorrectly logged for web sessions)

klizhentas commented:
retest this please
